### ISSUE TITLE
Any chance of Alexa multiroom audio support?

### ISSUE THREAD

**Issue by stingray072** [kalkih/mini-media-player#195]
Sonos multiroom looks so slick! Any chance of Alexa multiroom support?

**Comment by kalkih** [comment]
Yeah it's pretty neat! 😃

That would be cool, but it requires the backend media player component to support the necessary multiroom / speaker group management actions, such as join group / unjoin group for me to support it in this card.

I'm not familiar with alexa but it seems to work similar to google cast where you set up static groups and give them a name? Depending on how that functionality work and whether it's accessible and manageable from their API it may or may not be possible to implement in the component.

Here's an issue open regarding multiroom in the alexa component project: https://github.com/custom-components/alexa_media_player/issues/78 but they don't seem to discuss management of speaker groups.

TLDR, if the functionality is added to the alexa component, I would gladly support it in this card.
